pr_check.sh

```diff
@@ -11,7 +11,7 @@ APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this compon
 #       git version of clowdapp.yaml(or any other) file from the pull request.
 COMPONENT_NAME="ccx-notification-writer ccx-notification-db-cleaner"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
 IMAGE="quay.io/cloudservices/ccx-notification-writer"
-COMPONENTS="ccx-data-pipeline ccx-insights-results dvo-writer ccx-smart-proxy ccx-notification-writer ccx-notification-service ccx-notification-db-cleaner notifications-backend notifications-aggregator notifications-engine insights-content-service ccx-mock-ams ccx-upgrades-sso-mock insights-content-template-renderer"  # space-separated list of components to load
+COMPONENTS="ccx-data-pipeline ccx-insights-results ccx-redis dvo-writer ccx-smart-proxy ccx-notification-writer ccx-notification-service ccx-notification-db-cleaner notifications-backend notifications-aggregator notifications-engine insights-content-service ccx-mock-ams ccx-upgrades-sso-mock insights-content-template-renderer"  # space-separated list of components to load
```

Comment:
why do we need smart-proxy, redis, etc.?

Reply:
Similar, see here: https://github.com/RedHatInsights/ccx-notification-service/pull/797#discussion_r1787329609

Reply:
As @matysek explained in the other pr : 
    - smart-proxy needs ccx-redis, otherwise it won't get deployed
    - iqe-tests require smart-proxy to check success of uploading a test archive.